src/components/Header.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
```

Comment:
#### Nitpick
Always comment your imports as done in the component above

Reply:
I have commented on all the imports